.golangci.yaml

```diff
@@ -0,0 +1,52 @@
+run:
+  timeout: 5m
+  skip-files:
+    - pkg/target/matchcrd_constant.go
+
+linters-settings:
+  gocritic:
+    enabled-tags:
+    - performance
+  gosec:
+    excludes:
+    - G108
+  importas:
+    no-unaliased: true
+    alias:
+      - pkg: "github.com/open-policy-agent/frameworks/constraint/pkg/client"
```

Comment:
do we need this for library?

Reply:
Hi @sozercan - Do we need anything else on this?  Thanks!

Reply:
oh i didn't realize library used this. resolving